src/pages/FoodItemsPage/FoodPage.jsx

```diff
@@ -0,0 +1,185 @@
+import React from "react";
+import { useState } from "react";
+import SearchBar from "../../components/SearchBar/SearchBar";
+import Category from "../../components/CategoryCard/Category";
+import Food from "../../components/Food-item-card/Food";
+function Page() {
```

Comment:
this should be FoodPage
```suggestion
function FoodPage() {
```